app/pkg/protocol/channel.go

```diff
@@ -60,7 +60,7 @@ func (channel *Channel) ApproveInitChannel(privateKey []byte) (state.Signature,
 
 // FundChannel deposits funds to already opened state channel.
 // It returns on-chain transaction with detailed information.
-func (channel *Channel) FundChannel(p Participant, privateKey []byte, opts ...gasprice.Station) (*types.Transaction, error) {
```

Comment:
`NewParticipant` returns pointer, so
use pointer in those kind of function

Reply:
all common functions will use a pointer to these structs `Contract`, `Participant`